app/services/course/assessment/question/programming_import_service.rb

```diff
@@ -0,0 +1,105 @@
+# Imports the provided programming package into the question. This evaluates the package to
+# obtain the set of tests, as well as extracts the templates from the package to be stored
+# together with the question.
+class Course::Assessment::Question::ProgrammingImportService
```

Comment:
Yes, that's right.

Not called yet because I haven't wrote it. I'm writing it now.